src/pages/login/login.html

```diff
@@ -53,4 +53,5 @@
     </div>
 
   </form>
+  <button ion-button (click)="goForgotPassword()" clear>Mot de passe oublié</button>
```

Comment:
this name is not really well chosen since you don't really make the page go anywhere in my opinion